specification/cognitiveservices/OpenAI.Inference/routes/files.tsp

```diff
@@ -0,0 +1,100 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-client-generator-core";
+import "../models/files.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using Azure.ClientGenerator.Core;
+
+namespace Azure.OpenAI;
+
+/**
+ * Gets a list of previously uploaded files.
+ *
+ * @param purpose A value that, when provided, limits list results to files matching the corresponding purpose.
+ * @returns The requested list of files.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-names" "mirrored API responds with a container"
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "not yet an Azure operation"
+#suppress "@azure-tools/typespec-azure-core/no-operation-id" "non-standard operations"
+#suppress "@azure-tools/typespec-azure-core/operation-missing-api-version" "not yet versioned"
```

Comment:
@trrwilson - some (maybe all?) of these file operations do need an apiVersion query parameter. Are we omitting it for a specific reason?

Reply:
(on reading another thread I see now that these aren't supposed to work, or, at least, it's purposefully different. So maybe these APIs aren't ready yet)

Reply:
The `file` and `batch` ops seem to not be available in Azure, I think that's why we are skipping attaching a version to this.